stable/drupal/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+- bitnami-bot
```

Comment:
I would say bitnami-bot should just be a reviewer, or not even in this list

Reply:
I can remove from the list